08-fractionCalculator/FracCalc.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+
+
+public class FracCalc {
+
+    /**
+     * Prompts user for input, passes that input to produceAnswer, then outputs the result.
+     * @param args - unused
+     */
+    public static void main(String[] args){
+        // TODO: Read the input from the user and call produceAnswer with an equation
+        // Checkpoint 1: Create a Scanner, read one line of input, pass that input to produceAnswer, print the result.
+        // Checkpoint 2: Accept user input multiple times.
+
+
+
+        Scanner input = new Scanner(System.in);
+        System.out.print("Please enter in your equation it must be in the format as first equation space operation space second equation. For example 4 + 3:  ");
+        String holdsEqu = input.nextLine();
+        String Input1 = produceAnswer(holdsEqu);
+        System.out.print(Input1);
+
+
+    }//end main method
+
+    /**
+     * produceAnswer - This function takes a String 'input' and produces the result.
+     * @param input - A fraction string that needs to be evaluated.  For your program, this will be the user input.
+     *      Example: input ==> "1/2 + 3/4"
+     * @return the result of the fraction after it has been calculated.
+     *      Example: return ==> "1_1/4"
+     */
+    public static String produceAnswer(String input){
+      int Space = input.indexOf(" ");//Here i am identifying the space in the equation. This will be used to find where my first num is located
+      int totalcha = input.length();//Length of the String
+      String firstEqu = input.substring(0, Space);//First Num
+      String Oper = input.substring(Space + 1, Space + 2);//The operator
+      String secondEqu = input.substring( Space + 2, totalcha);//Second Num
+      int check1 = firstEqu.indexOf("_");//The _ in the fraction for example 4_5/6 for the first fraction
+      int check2 = secondEqu.indexOf("_");// Does the samething as line 36 but for the second fraction
+      int Slash1 = firstEqu.indexOf('/');//The slash in the fraction
+      int Slash2 = secondEqu.indexOf("/");// The slash in the second fraction
+      String allParts;
+
+
+
+      if (check1 != -1){//If check is -1 that means there is _ in the fraction
+        String wholeNum = firstEqu.substring(0, check1);//This takes the whole num from fraction one
+        String num = firstEqu.substring(check1 + 1, Slash1);//Takes the numerator from fraction one
+        String den = firstEqu.substring(Slash1 + 1);//Takes the denominator from fraction one
+        System.out.println("the whole number is: " + wholeNum);
+        System.out.println("The numerator is; " + num );
+        System.out.println("The denominator is: " + den);
+      }
+      else{//If there is no _ in the fraction it prints the first/second num as it is
+        String wholeNum = firstEqu;
+        System.out.print("Number one is: " + wholeNum);
```

Comment:
I guess I'm a bit confused on the output of this... Example: I typed in  4/3 + 5/6 and the output was 
`Number one is: 4/3 5/6
 5/6`